### PR TITLE
odf: add bugzilla rules and approval checks for volume-replication-operator

### PR DESCRIPTION
**Pull request by agarwal-mudit**
Signed-off-by: Mudit Agarwal <muagarwa@redhat.com>

### CODE REVIEW

File: core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+tide:
+  queries:
+  - excludedBranches:
+    - devel
```
Comment (Madhu-1) [inline core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml:4]: it should be devel or main? for volume replication we use `main` branch we don't have `devel` branch

File: core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - needs-rebase
+    repos:
+    - red-hat-storage/volume-replication-operator
+  - includedBranches:
+    - devel
```
Comment (Madhu-1) [inline core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml:19]: same here.

File: core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml (hunk 1 above)
Comment (agarwal-mudit) [reply core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml]: fixed

File: core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml (hunk 2 above)
Comment (agarwal-mudit) [reply core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml]: fixed

**Pull request merged**